core/server/web/api/v2/admin/routes.js

```diff
@@ -4,13 +4,14 @@ const multer = require('multer');
 const api = require('../../../../api');
 const apiv2 = require('../../../../api/v2');
 const mw = require('./middleware');
+const upload = require('./uploadHelper');
 
 const auth = require('../../../../services/auth');
 const shared = require('../../../shared');
 
 // Handling uploads & imports
-const tmpdir = os.tmpdir;
-const upload = multer({dest: tmpdir()});
+
+upload.multerInstance(multer({dest: os.tmpdir()}));
```

Comment:
Think if mutler is going to be used only once and specifically to initialize the uploadHelper, it would make sense to move this whole initialization logic into helper :thinking:

Reply:
Yes, the wrapper was just an inital thought, I tried to change as little as possible on the existing files of Ghost just in case! Great then, I’ll just pass os.tmpdir() as an argument.